src/main/java/org/openelisglobal/menu/applicationpropertiescontroller/ApplicationPropertiesController.java

```diff
@@ -0,0 +1,38 @@
+package org.openelisglobal.menu.applicationpropertiescontroller;
+
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.http.MediaType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Map;
+import java.util.HashMap;  
+
+@RestController
+@RequestMapping("/rest/properties")
+public class ApplicationPropertiesController {
+
+    @Autowired
+    private ConfigurableEnvironment env;
+
+    @GetMapping
+    public ResponseEntity<Map<String,String>> getProperties() {
+        Map<String,String> properties = Map.of(
+            "fixedRate" , env.getProperty("facilitylist.schedule.fixedRate")
```

Comment:
Only hard code the prpreties here , and just dynamically fetch them in the UI without hard-coding them again in the UI

Reply:
I think its also better to use the actual property name `facilitylist.schedule.fixedRate` as a key instead of `fixedRate` so that you dont have to re write again when building the json file on submit. it should dynamically be built in the UI without re hardcoding any property again